iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.hpp

```diff
@@ -64,11 +64,12 @@ class RelativePointer : public BaseRelativePointer
     template <typename U = T>
     typename std::enable_if<!std::is_void<U>::value, const U&>::type operator*() const noexcept;
 
-    /// @brief read-only access to the underlying object
+    /// @brief read-only access to the underlying object. If the RelativePointer does not point to anything the
+    /// application terminates.
     /// @return a pointer to the underlying object
     T* operator->() const noexcept;
 
-    /// @brief access the underlying object
+    /// @brief access the underlying object.
```

Comment:
Actually 

```suggestion
    /// @brief Access the underlying object.
```

but this is inconsistent with the other documentation so we should not change it.

Reply:
@MatthiasKillat I capitalized all sentences in the beginning.